fractions/microprofile/microprofile-restclient/README.adoc

```diff
@@ -0,0 +1,42 @@
+= MicroProfile Rest Client
+
+== CDI Interceptors Support
+
+In general, Rest Client proxies are not created by the CDI container and therefore method invocations do not pass through CDI interceptors.
+In Thorntail, however, you can associate business method interceptors (denoted by `@AroundInvoke` annotation) with a Rest Client proxy.
+You can do so by using interceptor bindings.
+This feature is non-portable.
+The primary use case is the support of xref:microprofile-fault-tolerance[] annotations, for example:
+
+[source,java]
+----
+import org.eclipse.microprofile.faulttolerance.Retry;
+
+@Path("/v1")
+interface MyClient {
+
+    @Retry(maxRetries = 3) // Retry on any exception thrown
+    @GET
+    @Path("/hello")
+    String hello();
+}
+----
+
+NOTE: The `org.eclipse.microprofile.faulttolerance.Asynchronous` is currently not supported because the underlying RESTEasy client is not able to handle the `java.util.concurrent.Future` return types.
+
+== RestClientProxy
+
+Additionaly, every Rest Client proxy implements `org.wildfly.swarm.microprofile.restclient.RestClientProxy` interface which allows to:
+
+* obtain the underlying `javax.ws.rs.client.Client` instance
+* release all associated resources, for example:
+
+[source,java]
```

Comment:
Please clarify what topic this piece of code pertains to.

Reply:
Ignore this, I misunderstood.

Reply:
See _for example:_ in the second bullet. I followed your recommendation ;-)

Reply:
Of course, thank you. I saw that later and hid my comments.